Resources/Prototypes/_Moffstation/PersonalItems/Wearables/catlord09-/Vortex_Vatia/floralmask.yml

```
@@ -0,0 +1,26 @@
+# Player catlord09- Character: Vortex Vatia
+
+- type: entity
+  parent: ClothingMaskBase
+  id: PersonalItemFloralMask
```

IDs for these follow a `PersonalItem{Character-Name}{Item-Short-Desc}` format.

imo
```suggestion
  id: PersonalItemVortexMask
```